src/lib/components/molecules/table/table.stories.jsx

```diff
@@ -61,21 +66,28 @@ export const Sortable = {
   },
 }
 
+const partyToColorMap = {
+  lab: 'lab',
+  con: 'con',
+  ld: 'libdem',
+  green: 'green',
+}
+
```

Comment:
do you think this kind of mapping helper function code has a place in this repo? If so perhaps we should pull this out into an easy to find shared helper file? (Doesn't have to be in this pull request).

Reply:
Good shout. The stories file is really just sample code, but I didn't need to use the mapping there at all. Will fix this by changing the data I pass to the table.